BlueprintUI/Sources/Element/Element.swift

```diff
@@ -56,11 +56,29 @@ public protocol Element {
     /// In Blueprint, elements that are displayed using a live `UIView` instance are referred to as "view-backed".
     /// Elements become view-backed by returning a `ViewDescription` value from this method.
     ///
-    /// - Parameter bounds: The bounds of this element after layout is complete.
-    /// - Parameter subtreeExtent: A rectangle in the local coordinate space that contains any children.
-    ///                            `subtreeExtent` will be nil if there are no children.
+    /// - Parameter context: The context this element is rendered in.
     ///
     /// - Returns: An optional `ViewDescription`.
-    func backingViewDescription(bounds: CGRect, subtreeExtent: CGRect?) -> ViewDescription?
+    func backingViewDescription(in context: ElementContext) -> ViewDescription?
```

Comment:
You're going to hate me (sorry); what do we think about:

```
func viewDescription(with context: ElementContext) -> ViewDescription?
```

Also, do we want `ElementContext`; or should it be `ViewDescriptionContext` or something?

Reply:
Both changes are fine with me! I had `with` originally but `in` seemed kinda nice with the one param? idk, fine with `with`

Reply:
(I think `ElementContext` is fine; just wondering if we'll ever want that name for something else)

Reply:
Already did `ViewDescriptionContext`